simple_https_client/src/main.rs

```diff
@@ -1,26 +1,19 @@
+//! A Simple TLS 1.3 HTTP Client Implementation
```

Comment:
The renaming seems unnecessary.

Reply:
The rename is required due to https://github.com/cryspen/bertie/pull/24/commits/c42f90fe60cf7b80414ab9587a91b0bb9914bff1 Otherwise Cargo can't know what binary to compile.

Alternative is to drop the rename. But in any case, having bin + lib seems not right for simple_https_client (under normal circumstances) and results in ... 

```
warning: bertie/simple_https_client/Cargo.toml: file found to be present in multiple build targets: bertie/simple_https_client/src/tls13client.rs
```

Is this how things should be? Maybe HACSPEC requires this, too?

Reply:
The lib there's a c&p error from the original toml. So that should certainly go.
But I'm not a big fan of the default file names (makes it impossible to find the right main.rs in a larger workspace).

Reply:
Okay, I will remove the lib section and revert the rename.